### ISSUE TITLE
FPS meter is not accurate

### ISSUE THREAD

**Issue by flackr** [magcius/noclip.website#25]
The FPS meter code is here:
https://github.com/magcius/noclip.website/blob/84fbfe7b852f9605530e5e479395debcc3c1623d/src/RenderStatistics.ts#L33

This code seems to assume that as soon as the script to produce the frame has finished running that the frame has been produced but there is a lot of work in the browser after javascript completes to draw the frame to the screen. On my laptop for example the site claims 300-500fps but chrome's FPS meter shows that I am only actually getting 30fps. This should be using the requestAnimationFrame timestamps to determine the effective framerate.

**Comment by magcius** [comment]
Agreed. The FPS meter was really for me to detect too much CPU usage during development. If you want help, patches very welcome -- I'm open to helping people get set up with this project.
